### PR TITLE
podcast and contact styling

### PR DESCRIPTION
**Pull request by agnieszkastanczuk**

### CODE REVIEW

File: src/components/PodcastPostLayout/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type Props = {
+  title: string;
+  description: string;
+  markdown: string;
+};
+
+const PodcastPostLayout = ({ title, description, markdown }: Props): JSX.Element => {
+    const components = {
+      h2: ({ children }: { children: any }) => (
```
Comment (mrruby) [inline src/components/PodcastPostLayout/index.tsx:12]: Can we change this any?

https://www.typescriptlang.org/docs/handbook/jsx.html

File: src/pages/contact.tsx [hunk 2]
```diff
@@ -18,8 +18,7 @@ const ContactPage = (): JSX.Element => {
     <Layout>
       <SEO title="Kontakt" />
       <MaxWithBgColorContainer extraStyle="pt-[50px]">
-      <Title>Kontakt</Title>
-      <Container
+      {/* <Container
```
Comment (mrruby) [inline src/pages/contact.tsx:21]: Can we remove this comments?

File: src/pages/contact.tsx [hunk 3]
```diff
@@ -33,7 +32,8 @@ const ContactPage = (): JSX.Element => {
             <StaticImage src={"../images/contact.webp"} alt="banner" />
           </Center>
         </Column>
-      </Container>
+      </Container> */}
```
Comment (mrruby) [inline src/pages/contact.tsx:35]: Can we remove this comments?

File: src/templates/podcast/podcast.tsx [hunk 4]
```diff
@@ -44,15 +48,21 @@ const PodcastPage = ({ data }: PodcastData): JSX.Element => {
   return (
     <Layout>
       <SEO title={title} description={description} />
-      <Title>{title}</Title>
+      <MaxWithBgColorContainer extraStyle="pt-[70px] px-3 lg:px-5 flex flex-col w-6xl max-w-[920px] mx-auto">
+      <Audio
+        url={`https://podcastada.s3.eu-central-1.amazonaws.com/Podcast_${season}_${episodeNumber}.mp3`}
+      />
+        <PodcastPostLayout title={title} description={description} markdown={rawMarkdownBody}  />
+        {/* <Title>{title}</Title>
```
Comment (mrruby) [inline src/templates/podcast/podcast.tsx:56]: Same here if it is what we want to have

File: src/templates/podcast/podcast.tsx [hunk 5]
```diff
@@ -44,15 +48,21 @@ const PodcastPage = ({ data }: PodcastData): JSX.Element => {
...
       <Description>{description}</Description>
       <Audio
         url={`https://podcastada.s3.eu-central-1.amazonaws.com/Podcast_${season}_${episodeNumber}.mp3`}
       />
       <div>
         <DesktopRowMobileColumn dangerouslySetInnerHTML={{ __html: html }} />
         <Disqus config={disqusConfig} />
-      </div>
+      </div> */}
```
Comment (mrruby) [inline src/templates/podcast/podcast.tsx:64]: and here

**Review by mrruby — CHANGES_REQUESTED** [review]

File: src/components/PodcastPostLayout/index.tsx (hunk 1 above)
Comment (agnieszkastanczuk) [reply src/components/PodcastPostLayout/index.tsx:12]: image

Can you suggest how I should write it? I don't think that's ok.

Comment (agnieszkastanczuk) [reply src/components/PodcastPostLayout/index.tsx:12]: image

Comment (mrruby) [reply src/components/PodcastPostLayout/index.tsx:12]:
```
  const components = {
    h2: ({ ...props }) => (
      <h2 className="font-bold md:text-adaSubtitle py-3" {...props} />
    ),
    h3: ({ ...props }) => <h3 className="font-bold py-3" {...props} />,
    h4: ({ ...props }) => <h4 className="font-bold py-3" {...props} />,
    ul: ({ ...props }) => <ul className="list-disc pl-6 pb-4" {...props} />,
    ol: ({ ...props }) => <ol className="list-decimal pl-6 pb-4" {...props} />,
    p: ({ ...props }) => <p className="pb-4" {...props} />,
  }
  ```
  
  How about now? Is this working?

Comment (agnieszkastanczuk) [reply src/components/PodcastPostLayout/index.tsx:12]: Yes, it's working !

Comment (mrruby) [reply src/components/PodcastPostLayout/index.tsx:12]: ufff

**Review by mrruby — APPROVED** [review]

**Pull request merged**